### ISSUE TITLE
Support Convolution&Batchnorm Fusing for Optimized Inference Mode

### ISSUE THREAD

**Issue by DonghakPark** [nnstreamer/nntrainer#2646]
One of the ways to accelerate running the NNTrainer in inference mode is to fuse operations. 
We are currently using this fusion when exporting to TensorFlow Lite.
By applying it to the current NNTrainer, we can improve the speed during inference.

Many Deeplearning Model using Batchnorm after Conv layer 

![img1 daumcdn](https://github.com/nnstreamer/nntrainer/assets/44962038/b43ff837-d9d3-41a2-8696-3c98aefda7b8)

and when inference we can fusing ops to below 

![img1 daumcdn](https://github.com/nnstreamer/nntrainer/assets/44962038/507ea2f2-298b-4669-a0ba-69948574fec3)
![img1 daumcdn](https://github.com/nnstreamer/nntrainer/assets/44962038/a7263570-cd89-4c85-881d-4e0258a3215f)

[image ref](https://treeinthepark.tistory.com/7)

**Comment by lhs8928** [comment]
Fusing the operation on infererence means that this graph is little bit different from the graph used to train the model.
To support fusing operation we might need to make 2 graphs for 1 model. (One for training and the other one for inference)

**Comment by DonghakPark** [comment]
> Fusing the operation on infererence means that this graph is little bit different from the graph used to train the model. To support fusing operation we might need to make 2 graphs for 1 model. (One for training and the other one for inference)

That's right !! when we save()&load() model we check [INFERENCE|TRAIN] Mode and make graph
